src/main/java/com/centralesupelec/chowchow/config/CorsFilter.java

```diff
@@ -0,0 +1,29 @@
+package com.centralesupelec.chowchow.config;
+
+import java.io.IOException;
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CorsFilter implements Filter {
+
+  @Override
+  public void init(FilterConfig filterConfig) throws ServletException {}
+
+  @Override
+  public void doFilter(
+      ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
+      throws IOException, ServletException {
+    HttpServletResponse response = (HttpServletResponse) servletResponse;
+    HttpServletRequest request = (HttpServletRequest) servletRequest;
+    String origin = request.getHeader("Origin");
+    response.setHeader("Access-Control-Allow-Origin", origin);
```

Comment:
Hacky!

Reply:
https://trello.com/c/9FZTcFTX/49-add-env-variable-for-cors-origin